editor/src/core/workers/ts/ts-worker.ts

```diff
@@ -498,6 +545,12 @@ function isNodeExported(node: TS.Node): boolean {
   )
 }
 
+const matcher = (importPath: string) => {
+  return [/\.avif$/, /\.bmp$/, /\.gif$/, /\.jpe?g$/, /\.png$/].some((regxp) =>
```

Comment:
This is duplicated from elsewhere where I made the comment about these being suffixes right? Would be nice to re-use that function or at least the array of values.

Reply:
Oops, this is left over duplicate code. Deleting.

Reply:
Fixed in 828a4a3